superset/connectors/sqla/views.py

```diff
@@ -377,10 +384,14 @@ class TableModelView(DatasourceModelView, DeleteMixin, YamlExportMixin):
         )
     }
 
-    def pre_add(self, table: "TableModelView") -> None:
+    def pre_add(  # pylint: disable=arguments-differ
```

Comment:
`pre_add(self, item: "TableModelView"): -> None` will this satisfy pylint?

Reply:
Oh nice, yeah, that resolves it. I thought it was complaining about the type annotation. Should have read closer.